raiden/api/v1/resources.py

```diff
@@ -199,3 +199,19 @@ def get(self):
         return self.rest_api.get_connection_managers_info(
             self.rest_api.raiden_api.raiden.default_registry.address,
         )
+
+
+class ChannelHistoryResource(BaseResource):
+
+    get_schema = EventRequestSchema()
+
+    @use_kwargs(get_schema, locations=('query',))
+    def get(self, token_address, partner_address=None, from_block=None, to_block=None):
```

Comment:
Would be nice to have type annotations as well.

Reply:
In resources.py no function has type annotations, should I add them to all of them?

Reply:
Just add it here for now, you can do the rest in another PR